core/Tracker/Model.php

```diff
@@ -375,9 +411,21 @@ private function visitFieldsToQuery($valuesToUpdate)
         return array($updateParts, $sqlBind);
     }
 
+    private function deleteDuplicateAction($newActionId)
```

Comment:
Which class do you mean? (couldn't see a method called `deleteDuplicateActions`)